test/test_utils/__init__.py

```diff
@@ -238,6 +241,16 @@ def get_dockerfile_path_for_image(image_uri):
 
 
 def get_expected_dockerfile_filename(device_type, image_uri):
+    if is_covered_by_e3_sm_split(image_uri):
```

Comment:
@shantanutrip it looks like there are a lot of references to "e3" in this PR, as mentioned in my general comment. Can you please go through and replace with EC2 anywhere you've added e3?

Reply:
Done